ted_sws/metadata_normaliser/resources/mapping_files_registry.py

```diff
@@ -1,4 +1,4 @@
-from ted_sws.metadata_normaliser.resources import get_mapping_file
+from ted_sws.metadata_normaliser.resources import get_mapping_json_file, get_mapping_csv_file
```

Comment:
I have a sense that The resources, will migrate into the "common" area because they are needed also in mappings. But let them be here for now.